apps/src/templates/teacherDashboard/sections.js

```diff
@@ -77,13 +79,22 @@ export function renderLoginTypeAndSharingControls(sectionId) {
   );
 }
 
-export function renderSectionTable(sectionId, loginType) {
+export function renderSectionTable(sectionId, loginType, courseId) {
   registerReducers({manageStudents, isRtl});
   const store = getStore();
 
   store.dispatch(setLoginType(loginType));
   store.dispatch(setSectionId(sectionId));
 
+  const showShareColumn = experiments.isEnabled(experiments.SHARE_COLUMN);
+
+  // Show share column by default for CSD and CSP courses
+  const CSDCourseId = 17;
+  const CSPCourseId = 18;
```

Comment:
I think you're instincts that hardcoding this seems less than ideal, but also that it's probably a fine in this case. However, I wonder if we should make this a hash so at least can be updated more easily in the future? `{
    CSDCourseId => 17, 
    CSPCourseId => 18,
    CourseWeDontKnowYet => 100
}
Then we could do a check like `hash.values.includes?(courseId)`. Or, rather, the JS equivalent of my Ruby-ish 🙂

Reply:
Good idea, I'll update.